app/decorators/user_decorator.rb

```diff
@@ -65,7 +65,7 @@ def confirmation_period
   end
 
   def masked_two_factor_phone_number
-    masked_number(MfaContext.new(user).phone_configurations.take&.phone)
+    MfaContext.new(user).phone_configurations.take&.masked_phone
```

Comment:
Just to confirm, since we're possibly returning `nil` here that we weren't previously. Is that okay?

**Edit:** Seems like same can apply for a few of the following revisions as well.

Reply:
- Looks like `masked_two_factor_phone_number` was completely unused, so I just removed it in eb82de413f5a0b663418e5bfef69e582ab4f07b5
- Checking the rest, I think they all keep the same empty string behavior, so I will leave them for now